nio_cli/commands/new.py

```diff
@@ -10,7 +10,18 @@ def __init__(self, options, *args, **kwargs):
         self._name = self.options['<project-name>']
 
     def run(self):
-        clone = "git clone git@github.com:{}/{}.git {}"
-        subprocess.call(
-            clone.format('nioinnovation', 'project_template', self._name),
-            shell=True)
+        clone = (
+            "git clone --depth=1 git@github.com:{}/{}.git {}"
```

Comment:
It's sort of just setting it up so that those could be swapped out for variables later.